int32 joint_id
---
bool is_successful